tools/schemacode/bidsschematools/render.py

```diff
@@ -499,36 +514,98 @@ def make_obj_table(subschema, field_info, src_path=None, tablefmt="github"):
         if isinstance(requirement_info, tuple):
             requirement_info, description_addendum = requirement_info
 
+        requirement_info = normalize_requirements(requirement_info)
         requirement_info = requirement_info.replace(
             "DEPRECATED",
             "[DEPRECATED](/02-common-principles.html#definitions)",
         )
 
         type_string = utils.resolve_metadata_type(subschema[field])
 
-        description = subschema[field]["description"] + " " + description_addendum
+        description = normalize_requirements(
+            subschema[field]["description"] + " " + description_addendum
+        )
 
         # Try to add info about valid values
         valid_values_str = utils.describe_valid_values(subschema[field])
         if valid_values_str:
             description += "\n\n\n\n" + valid_values_str
 
-        # A backslash before a newline means continue a string
-        description = description.replace("\\\n", "")
-        # Two newlines should be respected
-        description = description.replace("\n\n", "<br>")
-        # Otherwise a newline corresponds to a space
-        description = description.replace("\n", " ")
         # Spec internal links need to be replaced
         description = description.replace("SPEC_ROOT", get_relpath(src_path))
 
-        df.loc[field_name] = [requirement_info, type_string, description]
+        df.loc[field_name] = [
+            normalize_breaks(requirement_info),
+            type_string,
+            normalize_breaks(description),
+        ]
 
     # Print it as markdown
     table_str = tabulate(df, headers="keys", tablefmt=tablefmt)
     return table_str
 
 
+def make_sidecar_table(schema, table_name, src_path=None, tablefmt="github"):
+    """Produce metadata table (markdown) based on requested fields.
+
+    Parameters
+    ----------
+    schema : dict
+        The BIDS schema.
+    table_name : qualified name in schema.rules.sidecars
+    src_path : str | None
+        The file where this macro is called, which may be explicitly provided
+        by the "page.file.src_path" variable.
+    tablefmt : string, optional
+        The target table format. The default is "github" (GitHub format).
+
+    Returns
+    -------
+    table_str : str
+        The tabulated table as a Markdown string.
+    """
+    if isinstance(table_name, str):
+        table_name = [table_name]
+    fields = {}
+    for table in table_name:
+        fields.update(schema.rules.sidecars[table].fields)
```

Comment:
May be worth printing on key collision, or having a rule for deciding which entry to keep.

Reply:
Changed it to an error.